backend/graphql/resolvers/Mutation.ts

```diff
@@ -1,9 +1,8 @@
 import { createBankAccountForUser, removeBankAccountById } from "../../database";
 
 const Mutation = {
-  createBankAccount: (obj: any, args: any, ctx: any) => {
-    const account = createBankAccountForUser(ctx.user.id!, args);
-    return account;
+  createBankAccount: (obj: any, args: any, ctx: { user: { id: string } }) => {
```

Comment:
I like the initiative here, but it might be better to define a shared `ResolverContext` type, e.g.

```
type ResolverContext = {
  user?: Express.User
}
```

that can be imported into the resolvers, and also passed as a context template type to `grapql-http`'s `createHandler` function - which will ensure that the `context()` property has the correct return type